astronomy-pipeline/db_connect.sh

```diff
@@ -0,0 +1,3 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
+psql -h $DB_HOST -p $DB_PORT -U $DB_USER $DB_NAME
```

Comment:
This is duplicated code maybe it would be more useful to have it connect to the astronomy table